### ISSUE TITLE
Admin Emails Unable to Send 

### ISSUE THREAD

**Issue by silverman63** [sailthru/sailthru-magento2-extension#43]
When trying to send an admin email like reset password (for admin), the following exception is raised:

```
{"0":"Notice: Undefined property: Magento\\Email\\Model\\BackendTemplate\\Interceptor::$templateVariables in .../sailthru-magento2-extension\/Mail\/TransportBuilder.php
```

Theoretically, adding a new DI config should fix this.

**Comment by robyerevan** [comment]
this happens when admin user tries to crate another admin user. Magento has to be in production mode to notice the crush.
 
Object type **\Magento\Email\Model\BackendTemplate** returned instead of 
**\Magento\Email\Model\Template** which does not have templateVariables property. 

see \Sailthru\MageSail\Mail\TransportBuilder::prepareMessage

Issue is easily silenced, but new admin user does not receive any email.

**Comment by silverman63** [comment]
@robyerevan this should be addressed by #53
